app/views/kaui/accounts/_form.html.erb

```diff
@@ -9,9 +9,14 @@
       </div>
     </div>
     <div class="form-group">
-      <%= f.label :first_name_length, 'First name length', :class => 'col-sm-3 control-label' %>
-      <div class="col-sm-9">
-        <%= f.text_field :first_name_length, :class => 'form-control' %>
+      <%= f.label :first_name_length, :class => 'col-sm-3 control-label' do %>
+          First name length<%= link_to '*', 'https://www.w3.org/International/questions/qa-personal-names',target: '_blank', rel: 'nofollow' %>
```

Comment:
The `*` is a bit confusing (it makes it look like it's mandatory). How about a Font Awesome question mark icon (see http://fontawesome.io/icon/question-circle-o/)?